csrankings-v.csv

```diff
@@ -66,7 +66,6 @@ Vangelis Metsis,Texas State University,http://cs.txstate.edu/~v_m137,7hDSfoQAAAA
 Vania Dimitrova,University of Leeds,https://eps.leeds.ac.uk/computing/staff/184/professor-vania-dimitrova,cDrHc2YAAAAJ
 Vania G. Dimitrova,University of Leeds,https://eps.leeds.ac.uk/computing/staff/184/professor-vania-dimitrova,cDrHc2YAAAAJ
 Varmo Vene,University of Tartu,http://www.ut.ee/en/read-why-varmo-vene-running-dean,AR7YQTUAAAAJ
-Varol Akman,Bilkent University,http://www.cs.bilkent.edu.tr/~akman,NOSCHOLARPAGE

```

Comment:
Web page still active; please point to change of affiliation.

Reply:
He is retired -> https://w3.cs.bilkent.edu.tr/en/retired-faculty-members/